test/metabase/query_processor/middleware/cache_test.clj

```diff
@@ -56,8 +56,12 @@
                  [hash (u/format-nanoseconds (.getNano (t/duration created (t/instant))))]))))
 
       i/CacheBackend
-      (cached-results [this query-hash max-age-seconds respond]
-        (let [hex-hash (codecs/bytes->hex query-hash)]
+      (cached-results [this query-hash strategy respond]
+        (assert (= :ttl (:type strategy)))
+        (assert (contains? strategy :avg-execution-ms))
+        (let [hex-hash (codecs/bytes->hex query-hash)
+              max-age-seconds (* (:multiplier strategy)
```

Comment:
This name only makes sense as long as the multiplier is 1000, right? I find this strange.

Reply:
Well this is awkward! It should be `max-age-ms`, of course. :)